composer.json

```diff
@@ -20,12 +20,17 @@
 		"issues": "https://github.com/Rarst/wps/issues"
 	},
 	"require"    : {
-		"php"                : ">=5.5.9",
-		"composer/installers": "~1.6",
+		"php": "^7.2 || ^8.0",
+		"composer/installers": "^v2.3.0",
```

Comment:
I've been thinking about just bumping this... But also pondering is there any downside to keeping v1 installers allowed (as OR)?

I mean if I could work on it right now, I'd review a whole plugin and bring it up to date. But since I don't/can't I don't really want to raise floor on PHP version without due diligence.

Reply:
I think my recommendation here is that if someone needs the older version of the installers, they can easily do so by limiting the version of the package that is installed. Bump this version as a breaking change, and tag it as such?

Reply:
Automattic/WP.org is [pushing PHP v7.4 as the minimum required](https://wordpress.org/about/requirements/) version. It's certainly possible to [run it in older environments](https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/)... To me version bumping, tagging and releasing under 2.x would both make sense and align with WP overall. 🤷‍♂️